modules/must-gather-flags.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="must-gather-flags_{context}"]
+= Must-gather flags
+
+The flags listed in the following table are available to use with the `oc adm must-gather` command.
+
+.{product-title} flags for `oc adm must-gather`
+[cols="1,1,3",options="header"]
+|====
+|Flag |Example command |Description
+
+|`--all-images`
+|`oc adm must-gather --all-images=false`
+|Collect `must-gather` data using the default image for all Operators on the cluster that are annotated with `operators.openshift.io/must-gather-image`.
+
+|`--dest-dir`
+|`oc adm must-gather --dest-dir='<directory_name>'`
+|Set a specific directory on the local machine to where the gathered data will be written.
+
+|`--host-network`
+|`oc adm must-gather --host-network=false`
+|Run `must-gather` pods as `hostNetwork: true`. Relevant if a specific command and image needs to capture host-level data.
+
+|`--image`
+|`oc adm must-gather --image=[<plugin_image>]`
+|Specify a `must-gather` plugin image to run. If not specified, {product-title}'s default `must-gather` image will be used.
```

Comment:
Is there a reason the second sentence of line 30 is mostly in italics? Not sure all of that needs to be italicized.

Reply:
It's not showing up as italicized in the preview, weird.

Reply:
That sometimes happens when there's a mismatch of single quotes. For some reason GitHub thinks it's italic. But it's not.

You can just ignore that.